src/views/RegisterPage/index.js

```diff
@@ -17,19 +25,39 @@ class RegisterPage extends Component {
       confirmpassword: undefined,
       termsandconditions: undefined
     };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
   }
 
   handleChange = event => {
-    const { info, value } = event.target;
-
-    this.setState({
-      [info]: value
-    });
+    this.setState({[event.target.name]: event.target.value});
     console.log(this.state);
   };
 
   handleSubmit(event) {
     event.preventDefault();
+    var attributeList = [];
+    var dataEmail ={
+      Name: 'email',
+      Value: this.state.email
+    };
+    var dataPersonalName ={
+      Name: 'name',
+      Value: this.state.firstname
+    };
+    var dataFamilyName = {
+      Name: 'family_name',
+      Value: this.state.lastname
+    };
+    attributeList.push(dataEmail);
+    attributeList.push(dataPersonalName);
+    attributeList.push(dataFamilyName);
+
+    UserPool.signUp(this.state.firstname, this.state.password, attributeList, null, (err, data) => {
+      if (err) console.error(err);
+      console.log(data);
```

Comment:
does cognito log the user in automatically after they're been signed up? if so, retrieve the JWT that's returned and put it in local storage, then redirect the user to /dashboard which should only be able to be accessed with a valid jwt. if the user isn't logged in automatically, it might make more sense to revisit this later once login has been figured out so don't worry about the details now

Reply:
I don't think the user is logged in automatically, that is something we would have to configure.